tested_requirements/requirements_35.txt

```diff
@@ -1,25 +1,26 @@
-asn1crypto==1.3.0
+# Generated on: Python 3.5.9
+# With snowflake-connector-python version: 2.3.0
```

Comment:
I changed how this files looks a bit, now customers should be able to use this as a requirements file straight away after downloading it.

Reply:
Except that it doesn't have `snowflake-connector-python` in the list, that's what this comment is for

Reply:
Just to confirm, this is to tell customers they need to have `snowflake-connector-python` already installed?

Reply:
Take a look at how it looked before: https://github.com/snowflakedb/snowflake-connector-python/blob/master/tested_requirements/requirements_35.txt
I wanted to make this easier to use for customers.
So now if you `wget` this file or copy-paste it, you can do `pip install -r file.txt` and it will work, but I don't think that it would work with the older files.
This just tells what version of the connector was used to create this requirements file.

Reply:
That sounds good!